openmetadata-ui/src/main/resources/ui/src/components/ExploreV1/ExploreV1.component.tsx

```diff
@@ -333,6 +341,45 @@ const ExploreV1: React.FC<ExploreProps> = ({
                         </Button>
                       </span>
                     </Col>
+                    {isElasticSearchIssue ? (
+                      <Col span={24}>
+                        <Alert
```

Comment:
Can't we move the entire alert to util ?

Reply:
As per the new mock we will be needed the layout created in getAlertBody everywhere, this alert with this text only need at single place. so there then moving whole alert I choose to move it's layout

Reply:
removed getAlertBody